web-api/openapi/document_management.yaml

```diff
@@ -31,7 +31,7 @@ paths:
       description: |-
         Add documents to the system. The system will create a representation of the document
         that will be used to match it against the preferences of a user.
-      operationId: ingestDocuments
+      operationId: createDocuments
```

Comment:
One linter requires `operationId` to start with a verb that identifies the type of the operation depending on which HTTP method we use. I think it is clearer this way and less likely to conflict with future API updates where we add new methods. But I don't have a strong opinion on this, so we can revert this commit.

Reply:
The only problem with `createDocuments` right now, is that you can just update existing documents via `createDocuments` as well.

Maybe `sendDocuments` or just `documents` as the POST method already implies a push?

Reply:
This linter only allows for `create` if you think it is a problem, we can disable it.
In this specific case, I think that, since we don't like too much that a document can be updated without a deletion, giving an API that doesn't encourage you to do it is good. Also, from our perspective, it is like you are putting in a new document since we are completely replacing the old one.